shared/chat/conversations-list/index.native.js

```diff
@@ -61,8 +61,8 @@ const Avatars = ({participants, youNeedToRekey, participantNeedToRekey, isMuted,
 const TopLine = ({hasUnread, showBold, participants, subColor, timestamp, usernameColor}) => {
   const boldOverride = showBold ? globalStyles.fontBold : null
   return (
-    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', maxHeight: 17, minHeight: 17}}>
-      <Box style={{...globalStyles.flexBoxRow, flex: 1, height: 17, position: 'relative'}}>
+    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', maxHeight: 18, minHeight: 18}}>
```

Comment:
i'm working on changing the styling of this whole component now so lets remove this